libs/core/overview/components/time-synchronization/time-synchronization-ids.enum.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export enum TimeSynchronizationIds {
+  SynchronizeButton = "synchronize-button",
```

Comment:
Please move this to the `e2e-test-ids` as a shared ID

```suggestion
  SynchronizeButton = "time-synchronaize-synchronize-button",
```